lib/util/got/index.js

```diff
@@ -0,0 +1,11 @@
+const got = require('got');
+const cacheGet = require('./cache-get');
+const renovateAgent = require('./renovate-agent');
+
+/*
+ * This is the default got instance for Renovate.
+ *  - Set the user agent to be Renovate
+ *  - Cache all GET requests for the lifetime of the repo
+ */
+
+module.exports = got.mergeInstances(renovateAgent, cacheGet);
```

Comment:
This is good for as long as the next instances after `renovateAgent` don't have the `user-agent` header (the instances are merged by order). So, if you use `got.extend` instead of `got.create` there, `got.mergeInstances(renovateAgent, cacheGet)` won't have any effect. But it's very easy to fix, just move `renovateAgent` to the end of the chain:

```js
got.mergeInstances(cacheGet, renovateAgent)
```

I suggest this change so you don't have to remember about moving it in case going for `got.extend` :)